src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSCloudPool.cs

```diff
@@ -445,15 +501,27 @@ public PSPoolStatistics Statistics
             }
         }
         
-        public System.Int32? TargetDedicated
+        public System.Int32? TargetDedicatedComputeNodes
```

Comment:
Same here.

Reply:
My intent is to just leave this as is (no obsolete) since this is a breaking change, unless otherwise suggested by one of the powershell maintainers.

Reply:
This is ok, but it needs to be documented in the upcoming breaking changes document,a s noted.

Reply:
Added documentation there